packages/react-core/src/components/Wizard/WizardNavItem.tsx

```diff
@@ -17,6 +17,8 @@ export interface WizardNavItemProps {
   onNavItemClick?: (step: number) => any;
   /** Component used to render WizardNavItem */
   navItemComponent?: React.ReactNode;
```

Comment:
do we support only button and a or anything goes? if the former we could limit it to those 2